apps/pie-microsite/src/_data/normalizeTokens.js

```diff
@@ -0,0 +1,29 @@
+const pieDesignTokens = require('@justeat/pie-design-tokens/dist/tokens.json');
+
+// clone tokens to avoid mutating the original reference.
+const clonedPieDesignTokens = JSON.parse(JSON.stringify(pieDesignTokens));
+
+/**
+ * normalizes the font alias tokens into wide/narrow groups
+ * @returns - alias tokens categorized by wide/narrow objects to be used in their respective pages.
+ */
+const normalizeFontAliasTokens = () => {
+    const compose = modifier => {
+        const tokens = JSON.parse(JSON.stringify(clonedPieDesignTokens.theme.jet.font.alias));
+        Object.keys(tokens).forEach(key => {
+            if (tokens[key].size) return;
+            tokens[key].size = tokens[key][`size--${modifier}`];
+        });
+        return tokens;
+    };
+
+    clonedPieDesignTokens.theme.jet.font.alias = {
+        wide: compose('wide'),
+        narrow: compose('narrow')
+    };
+};
+
+// normalization steps
+normalizeFontAliasTokens();
+
+module.exports = clonedPieDesignTokens;
```

Comment:
question here - if the module.exports was just to return `normalizeFontAliasTokens()` (provided the function returned the new object), would that only be called once during the build or multiple times?

If it's only called once, it might be cleaner to go with that approach rather than calling the function in the module outside of any function but this is just personal preference.

Reply:
yes perfect - that's what I was thinking and poorly explained! :)